hazelcast-client/src/main/java/com/hazelcast/client/connection/nio/ClusterConnector.java

```diff
@@ -133,7 +133,7 @@ private Connection connectAsOwner(Address address) {
         Connection connection = null;
         try {
             logger.info("Trying to connect to " + address + " as owner member");
-            connection = connectionManager.getOrConnect(address, true);
+            connection = connectionManager.getOrConnect0(address, true);
```

Comment:
Shows the problem with the current approach. There should not be a coupling to an internal method.